gilbreth_grasp_planning/config/tool_plan_param.yaml

```diff
@@ -1,38 +1,60 @@
-parameters:
-  lift_distance: 0.3
-  drop_distance: 0.3
-  conveyor_speed: 0.2
+env_param:
+  lift_distance: 0.1
+  drop_distance: 0.2
+  conveyor_velocity: 0.2
+  conveyor_power: 0.5
   conveyor_belt_length: 2.8
 
-pose_timers:
-  approach_time: 11.0
-  pick_time: 13.5
-  retreat_time: 15.0
-  place_time: 20.0
+pose_timer:
+  approach_time: -3.0
+  pick_time: 0
+  retreat_time: 3.0
+  place_time: 7.0
 
-bin_origins:
+pick_offset:
```

Comment:
The pick point on the top surface is a little bit higher. When the gripper reach the pick pose to grasp the object, there is extra space between the vacuum gripper and the object's top surface. Therefore, the vacuum gripper can not grasp the object. There is a config file in perception package, which controls the output pick pose. @lianjunli said that it would be better not to change that config file, but to use offset to reduce the z-direction pick position. Last time I changed the config file in perception node (not the new perception code, but it also includes ICP to stabilize the result), the output will be unstable, especially for gear. I can run some test to check if the new perception code solve these problem.